website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -17,7 +17,7 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 dbt Labs plans to release dbt Core v1.8.0-b1 and dbt Labs-maintained adapters on February 28th, 2024.
 
-## Keep on latest version <Lifecycle status='public preview' />
+## Keep on latest version 
 
 With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud) setting for more details.
 
```

Comment:
Can you remove the callout about "Availability" below? This is now supported for all adapters